script.js

```diff
@@ -1,11 +1,66 @@
+// let lyrics;
+// let wordList;
+// function preload() {
+//   //load our assets
+//   lyrics = loadStrings('./assets/lyrics.txt');
+//   console.log(lyrics);
+// }
+
+// function setup() {
+//   createCanvas(windowWidth, windowHeight);
+//   background(250);
+
+//   //how do we go from text file to list of words, list of lines to list of words
+//   // for each line inside lyrics, run this split code
+//   //save tjpse wprds omtp tje wordlist
+//   lyrics.forEach((line) => {
+//     wordList.push(...line.split(' '));
+//   });
+
+//   console.log(wordList);
+
+// }
+
+// function draw() {
+//   // let currWord = "hello";
+//   // text(currWord, mouseX, mouseY);
+//   ellipse(mouseX, mouseY, 50, 50);
+
+// }
+
+let backgroungImg;
+let linesList = [];
+let wordsList = [];
+let stroke_color; 
+let fill_color; 
```

Comment:
you don't need `fill_color` here since you recalculate every frame in the draw function.